### DISCUSSION TITLE
Fehlermeldung nach Update

### DISCUSSION THREAD

**Discussion by heliuli** [community/community#129797] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Hallo,
Ich nutze evcc seit 3 monate ohne probleme unter linux Mint ohne Probleme .
Heute 25.06.2024  ca 08.00 Uhr Systemaktualisierung durchgeführt .
danach erscheint : " Error during startup. Check your configuration and restart.
cannot create vehicle 'ev5': cannot create vehicle type 'template': cannot create vehicle type 'custom': decoding failed due to the following error(s): '' has invalid keys: vin"

ich habe weder die yaml datei verändert, sonst noch irgendwas am System verändert.

Bitte helft mir , mit " einfachen" Lösungen.

Bisherige Aktualisierungen  des Systems ,haben immer funktioniert.

Gruss Marco

**Comment by queenofcorgis** [comment]
Hi @heliuli , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be  closing and locking any posts in languages other than English for now, including this one.

Please note for future posts: The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.
